In browser, printing -Infinity displays null

If you run this code at webppl.org, you get unexpected behavior:

```
var model = function() {
  return randomInteger(2);
};

var myERP = Rejection(model, 100);

print(myERP.score([], 3)) // => null
print(typeof myERP.score([], 3)) // => "number"
print(myERP.score([], 3) == null) // => false
print(myERP.score([], 3) == "null") // => false
```

This means models run in the browser don't properly handle factor statements for elements not in the support (``Uncaught AssertionError: Number expected.``)

The weirdest part is that the score returns null, but it doesn't behave like a null value! So there's no way to catch out-of-support elements. 

This code runs properly on my local machine for both dev and master branches, so I'm not sure what the fix would be.

--- a/assets/js/custom.js
+++ b/assets/js/custom.js
@@ -38,7 +38,7 @@ function jsPrint(x){
   } else {
     resultDiv.append(
       document.createTextNode(
-        (typeof x == 'string' ? x : JSON.stringify(x)) + "\n"));
+        (typeof x == 'string' ? x : util.serialize(x)) + "\n"));
   }
 }
 
@@ -209,7 +209,7 @@ function webpplObjectToText(x){
   if (isErp(x)){
     return "<erp>";
   } else {
-    return JSON.stringify(x);
+    return util.serialize(x);
   }
 }
 

